### ISSUE TITLE
Fetch the headers only

### ISSUE THREAD

**Issue by SumatraPeter** [aria2/aria2#881]
[**cURL**](https://curl.haxx.se) has an option [**-I or --head**](https://curl.haxx.se/docs/manpage.html#-I) which works as follows:

> (HTTP FTP FILE) Fetch the headers only! HTTP-servers feature the command HEAD which this uses to get nothing but the header of a document. When used on an FTP or FILE file, curl displays the file size and last modification time only.

As far as I can tell **aria2** doesn't have an equivalent option. Can it be added? Thanks!

**Comment by tatsuhiro-t** [comment]
The closest feature aria2 already have is --dry-run.

**Comment by SumatraPeter** [comment]
Do you mean one has to use --dry-run and then check the log for the headers? Would it be possible to have a similar (less cumbersome) feature that returns only the headers?

**Comment by tatsuhiro-t** [comment]
Yes.  I rather recommend to use curl for this kind of stuff.  aria2 is just a tool to download some files, not a something to inspect protocol stuff.
